source/PreferencesPanel.cpp

```diff
@@ -81,6 +82,10 @@ namespace {
 	const string ALERT_INDICATOR = "Alert indicator";
 	const string HUD_SHIP_OUTLINES = "Ship outlines in HUD";
 
+	const map<Command, string> UNLOCKABLE_CONTROLS = {
+		{Command::CLOAK, "found cloaking"}
+	};
```

Comment:
Not a fan of the name "UNLOCKABLE".
It is not locked. The idea may be that it is hidden until the player actually has a use for it, but that doesn't mean it is locked. We wouldn't say the afterburner control is locked when the player doesn't have an afterburner even though the control does nothing for them in that case. Or that the escort command controls are locked when the player doesn't have escorts.
This label is being hidden to avoid spoiling something. I think the name should reflect that.

Reply:
SPOILER_CONTROLS?

Reply:
Should TCs be able to define their own sets of controls for, say, "eat apple"? The mapping between property name, UI label, and activation control should be defined in a data file.